src/components/SearchForm.js

```diff
@@ -0,0 +1,74 @@
+import { useContext, useState } from 'react';
+import { GoogleMapsContext } from '../context/GoogleMapsContext';
+import { MapCenterContext } from '../context/MapCenterContext';
+import planetSearch from '../assets/search-location-solid.svg';
+
+export const SearchForm = () => {
+  const [query, setQuery] = useState('');
+  const valueGoogleMapsAPI = useContext(GoogleMapsContext);
+  const valueMapCenter = useContext(MapCenterContext);
+  const { map, maps } = valueGoogleMapsAPI;
```

Comment:
`map` is the map --- the container. `maps` is an object that you can use to access/interact with the places API.

Reply:
Just for my understanding ^ 👍